README.md

```diff
@@ -11,15 +11,18 @@ StyleKit is a iOS library that styles native controls with a CSS like JSON forma
 
 `github "dmgctrl/StyleKit"`
 
-## Install StyleKit in New Project
+## Install StyleKit as a submodule
 
 1. ` cd ` Into the root of the project you'd like to install StyleKit
-2. add styleKit submodule in ` Libraries/ ` directory ` git submodule add https://github.com/dmgctrl/StyleKit.git Libraries/StyleKit `
-3. create ` Style/ ` in the ` ${TARGET_NAME}/ ` directory of your project
-4. add your ` Style.json ` file to the ` Style/ ` directory
-5. Open Xcode project and navigate to your targets ` Build Phases `
-6. Add a new ` Run Script ` under the ` Target Dependencies ` build phase
-7. Add the following BASH Script :
+2. Add styleKit submodule in ` Libraries/ ` directory ` git submodule add https://github.com/dmgctrl/StyleKit.git Libraries/StyleKit
+
+## Configure StyleKit
+
+1. create ` Style/ ` in the ` ${TARGET_NAME}/ ` directory of your project
+2. add your ` Style.json ` file to the ` Style/ ` directory
```

Comment:
Since you changed the name of the input/output file, we should probably update those names here also.